src/database/index.tsx

```diff
@@ -90,6 +91,10 @@ const getDb = async () => {
 };
 
 const buildCreatePayload = (entity: any) => {
+  // Make sure the 'id' key doesn't exist on the entity object
+  // or else Firestore won't permit the addition.
+  delete entity.id;
```

Comment:
Just moving this up here so we don't have to remember to make this `delete` call every time we add a new model of any type